pkg/controllers/machinesetsync/machineset_sync_controller.go

```diff
@@ -98,88 +106,243 @@ func (r *MachineSetSyncReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return nil
 }
 
-// Reconcile reconciles CAPI and MAPI machines for their respective namespaces.
-//
-//nolint:funlen
+// Reconcile reconciles CAPI and MAPI MachineSets for their respective namespaces.
 func (r *MachineSetSyncReconciler) Reconcile(ctx context.Context, req reconcile.Request) (ctrl.Result, error) {
-	logger := log.FromContext(ctx, "namespace", req.Namespace, "name", req.Name)
+	logger := log.FromContext(ctx).WithValues("namespace", req.Namespace, "name", req.Name)
+	ctx = logr.NewContext(ctx, logger)
 
-	logger.V(1).Info("Reconciling machineset")
-	defer logger.V(1).Info("Finished reconciling machineset")
+	logger.V(1).Info("Reconciling MachineSet")
+	defer logger.V(1).Info("Finished reconciling MachineSet")
```

Comment:
Do we want to have a conversation about consistency between capitalising these keywords?

Reply:
I think I've tried to favour `MachineSet` over `machineset` in this. 

Happy to revert and favour `machineset` though.

Reply:
CC @damdo any strong opinions?

I think for CPMS we were using the style of the product docs, so we always wrote out `control plane machine set`, maybe we want to follow suit here with that too?

Reply:
Hmm yep if we are using that elsewhere then we should do the same here!